SpecFlowProjectMars1/StepDefinitions/ManageLanguagesStepDefinitions.cs

```diff
@@ -0,0 +1,65 @@
+using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium;
+using SpecFlowProjectMars1.Pages;
+using System;
+using System.Reflection.Emit;
+using TechTalk.SpecFlow;
+using SpecFlowProjectMars1.Utilities;
+
+namespace SpecFlowProjectMars1.StepDefinitions
+{
+    [Binding]
+    public class ManageLanguagesStepDefinitions:CommonDriver
+    {
+        [Given(@"the language ""([^""]*)"" with level ""([^""]*)"" is already present")]
+        public void GivenTheLanguageWithLevelIsAlreadyPresent(string language, string level)
+        {
+            WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(10));
+
+            // Navigate to the languages section if not already there
+            IWebElement languageSectionButton = driver.FindElement(By.XPath("//a[contains(text(),'Languages')]"));
```

Comment:
you steps should not have any webelements, they should be kept in pages

Reply:
missed to remove this file..it was not used anywhere..sorry will remove it